posydon/grids/scrubbing.py

```diff
@@ -152,3 +153,101 @@ def keep_after_RLO(bh, h1, h2):
         new_h2["star_age"] -= age_to_remove
 
     return new_bh, new_h1, new_h2
+
+
+def keep_till_He_depletion(bh, h1, h2, Ystop=1.0e-5, XCstop=1.0):
```

Comment:
My main comment is that I am not sure that the  "He depletion" term in the 'stop_at_He_depletion' and 'keep_till_He_depletion' is the most appropriate term, as in the case THRESHOLD_CENTRAL_ABUNDANCE  = Xc<~0.3-0.4, we probably go BEYONG He depletion as we have almost depleted all Carbon in the center too. We used to use another term, sth like "Close_to_carbon_depletion", do you think it makes sense here?

Reply:
Yes, the name was taken when we were thinking, that we should get all the way back to He-deletion. With later changes it turns out, that the name might not be that well anymore.

Hence, I'd more go to change it to what it is: the function keeps stuff till both requirements are meet, hence I suggest `keep_till_central_abundance_He_C`, the function is aimed to be more general, that one can use it for any combination of He and C abundance specified.
For the flag, we can take something like `stop_before_carbon_depletion`, because otherwise it would stop at carbon depletion, hence the name should represent the difference to that.
Would you agree with that?